### PR TITLE
Final Version of the NewsWeek Clone

### PR DESCRIPTION
**Pull request by SpaYco**
## Building with Backgrounds and Gradients: Apple Clone
- Added a Navigation Bar 
- Added Main Page
- Added a Footer
- Used Bootstrap classes
- No linter issue
- No HTML issues

[Live Version](https://raw.githack.com/Spayco/NewsWeek-Clone/feature/index.html)

**PLEASE NOTE:** my commits are done in the same 10 minutes because I accidentally removed all commits from my 'beta' branch (GitHub refused the pull request, and google said do rebase, but it caused all this), so I had to recopy everything to a new branch, you can `git reflog` to see all previous commits that have been deleted

### CODE REVIEW

File: Media/main.css [hunk 1]
```diff
@@ -0,0 +1,752 @@
...
+  max-width: 1350px;
+  margin: 0 auto;
+  overflow-x: hidden;
+}
+
+/* a fix for .row, it uses -15 on margin left and right, which causes problems */
+.row {
+  margin-left: 0;
+  margin-right: 0;
+}
```
Comment (euqueme) [inline Media/main.css]: please don't use this fix (from line #14 to #18) :no_good_woman:  instead use bootstrap classes for **margin-left: 0**; and **margin-right: 0**; every where you use the **.row** class in yor HTML file

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request Under the review by Maria Eugenia Queme P. (TSE)

### Hello @Spayco :handshake: 

### Your page looks very similar to the original and that deserves recognition :medal_sports: there are some key changes to make still :key: :construction_worker_man: 

## Small screens

**Section Positioning**

- [x] please follow the structure for the first 4 sections in this screen size:

1. TOP STORY
2. MORE STORIES
3. CULTURE & TRAVEL
4. FEATURED STORIES

- Yours

image

- Original

image

**MORE STORIES**

- [x] the articles have to take 60% of the screen width and the title and content only 40%

- Yours

image

- Original

image

**LATEST NEWS**

- [x] the articles have to take 60% of the screen width and the title and content only 40%

- Yours

image

- Original

image

**IN THE MAGAZINE**

- [x] the IN THE MAGAZINE title is overlapping with the preview magazine lets fix it!

- Yours

image

- Original

image

- [x] The articles have to take 100% of the screen's width, yours seem to have big white spaces on the sides, also they should have some more spacing in between them

- Yours

image

- Original

image

**U.S, World, Business, Tech & Science, Culture and Sports**

- [x] The image of the aside articles of these sections should take 60% of the screen width and the headline only 40%

- Yours

image

- Original

image

**SUBSCRIBE**

- [x] the preview magazines should be centered and the headline, paragraph, and button should come after and also horizontally centered

- Yours 

image

- Original

image

## Medium Screens

**Section Positioning**

- [x] please follow the structure of the 2 columns for the first 4 sections in this screen size:

         LEFT                                                           RIGHT
1. FEATURE STORIES                          |   TOP STORY
2. FEATURE STORIES (more articles)  |   CULTURE & TRAVEL
3. SPONSORED INSIGHT                     |   MORE STORIES

- Yours

image

- Original

image

## Big Screens

**Editor's Pick**

- [x] The articles should be split into 4 columns instead of 2

- Yours

image

- Original

image

## All sizes

**Footer**

- [x] remove the text "git" on the top of the NEWSWEEK logo

- Yours
image

- Original
image

## Bootstrap usage

- [x] This project is all about using bootstrap, use bootstrap everywhere for the new changes and replace these selectors in your CSS file

- padding: 0; (12 ocurrences )
- padding-left: 0; (1 ocurrences )
- padding-right: 0; (1 ocurrences )
- margin: 0; (8 ocurrences )
- margin-left: 0; (4 ocurrences )
- margin-right: 0 (2 ocurrences )
- margin-top: 0 (2 ocurrences )
- margin-bottom: 0 (1 ocurrences )
- color: black; (8 ocurrences in yor CSS file)
- color: white; (10 ocurrences )
- text-align: center; (2 ocurrences )
- background-color: white; (4 ocurrence )
- display: none; (1 ocurrences )
- text-decoration: none; (15 ocurrences)
- font-weight (28 occurrences, strongly recommend not using the number values and use the bootstrap classes instead)

### That's it! :raised_hands: 

### Please :pray:  do these changes :memo: and submit back for a code review when you are done :rocket: 

Happy coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

File: Media/main.css (hunk 1 above)
Comment (SpaYco) [reply Media/main.css]: done!

**Comment by SpaYco** [comment]
hey, i fixed everything you mentioned, hopefully i didn't miss something! 
one note though, you might find to `margin: 0;` because i used `margin: 0 auto;` to center, i googled and didn't find any class like it on bootstrap

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Spayco!  
### Your project looks :+1: great!
But still next parts look not like original:
* [x] Small version:
image
Original:
image
* [Optional] Take a look how each section is aligned after another. Your project will look much better if you will align all sections properly:
image
Original:
image
* [ ] Tablet version:
Next part looks weird, that needs to be fixed:
image

Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @Spayco       
#### Great Job so far, but there is one  more thing required. 
### General
###### Small Screen
- [x] The ***Top Story Section***  should look more like the original (Give it padding)  & the ***Sign in & Subscribe button*** should be positioned well.
Your Version


Original Version



.



 

### Make changes and submit another code review request.
*Happy coding*

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Under Review By Tolulope Olusakin(TSE)
Please check on this in a few minutes for some feedback

**Comment by nikasulo** [comment]
# Great job building this @Spayco!
It's all coming together pretty nicely and you've worked hard to try to implement the requested changes :ok_hand: Please consider my comments below along with the attached screenshots on the changes I'm proposing

#### Responsiveness
The major issues with the project currently, revolve around responsiveness and the following sections need a thorough inspection:

* Navigation
Please consider looking into the navigation bar on mobile screens, it's a bit disorganized at the moment, I've attached a screenshot below, please compare yours with the original at 320px
screenshot
screenshot

* Top Story
Please consider looking into the padding in the top story section of the page, compare yours with the original page's and you may find that it isn't supposed to sit smack on the edges of the browser pane on mobile screens
screenshot

* Footer
Please consider the following screenshot in conjunction with what is present on the original site, you may find that there needs to be a little more space between the Newsweek logo and social icons
screenshot

* Subscribe Section
Please consider the following section in the screenshot below, comparing it with that of the original page, you may find some discrepancies in the responsiveness of the image. Please consider making the appropriate changes
screenshot

### Summary
You have done a great job building this out, please consider my comments above, make the required changes and request another review. I wish you the best in all your endeavors! :rocket: See you at the top :medal_military: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by SpaYco** [comment]
fixed all the issues, 
if there's anything else that I need to change, please let me know, applying for another review

**Review by nikasulo — APPROVED** [review]
# Really awesome job! @Spayco 
You may merge this and keep going :handshake: 

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Pull request merged**